<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyColorDialog</class>
 <widget class="QDialog" name="MyColorDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>620</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_HueSaSel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 186, 186);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_showColor">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>5</y>
     <width>60</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 186, 186);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comb_RGBToHSV">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_colstr">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>晚歌自制夏天的风</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>色值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_RGBToHSV">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>晚歌自制夏天的风</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>转换</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_RText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>红 色</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_GText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>绿 色</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBox_baseColor">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>305</y>
     <width>320</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54,54,54);
color: rgb(161, 0, 255);
border-radius: 5px;</string>
   </property>
   <property name="title">
    <string>基本色块</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBox_custColBlo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>305</y>
     <width>300</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54,54,54);
color: rgb(161, 0, 255);
border-radius: 5px;</string>
   </property>
   <property name="title">
    <string>自定义色块</string>
   </property>
   <widget class="QPushButton" name="btn_addColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_removeColor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>移除</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_BText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>蓝 色</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_lightValue">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>264</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ligntness">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>5</y>
     <width>20</width>
     <height>255</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 186, 186);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_R">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_G">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_B">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_colHStr">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
color: rgb(255, 255, 255);
border-radius: 5px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_A">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_AText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>255</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 5px;
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MyColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MyColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
